jobs/list.ts

```diff
@@ -33,4 +34,6 @@ export const allJobs: CSCronJob[] = [
     { cronTime: '00 */15 * * * *', jobFunction: syncToWebflow, name: 'syncToWebflow' },
     // Send Slack Messages monthly:
     { cronTime: '00 00 10 01 * *', jobFunction: postStatisticsToSlack, name: 'postStatisticsToSlack' },
+    // Every night, trigger actions for courses that ended yesterday
+    { cronTime: '00 00 03 * * *', jobFunction: notificationsEndedYesterday, name: 'notificationsEndedYesterday' },
```

Comment:
I guess we should ask Tobi whether we should rather send this out at daytime (in Western Europe)